### PR TITLE
Add support for mysql backup and restore

### PR DESCRIPTION
**Pull request by vishalanarase**
```bash
❯ ./civo --config staging.civo.json db bk ls psql
Scheduled backup
+-------------+---------------+------------+-----------+-------------+------------------+
| Database ID | Database Name | Software   | Schedule  | Backup Name | Backup           |
+-------------+---------------+------------+-----------+-------------+------------------+
| f41a3f      | psql          | PostgreSQL | 0 0 0 * * | update-test | 20240131-000009F |
+-------------+---------------+------------+-----------+-------------+------------------+
Manual backups
+-------------+---------------+------------+------------------+
| Database ID | Database Name | Software   | Backup           |
+-------------+---------------+------------+------------------+
| f41a3f      | psql          | PostgreSQL | 20240129-092223F |
+-------------+---------------+------------+------------------+
| f41a3f      | psql          | PostgreSQL | 20240129-093432F |
+-------------+---------------+------------+------------------+
| f41a3f      | psql          | PostgreSQL | 20240129-095942F |
+-------------+---------------+------------+------------------+
  cli   mysql-manual-backup-restore !9                                                                                                  3s  02:08:58 PM
❯ ./civo --config staging.civo.json db bk ls mysql
Manual backups
+-------------+---------------+-----------+---------------+----------+--------+
| Database ID | Database Name | Backup ID | Backup Name   | Software | Status |
+-------------+---------------+-----------+---------------+----------+--------+
|      599730 | mysql         | 2daa05    | mysql-backup2 | MySQL    | FAILED |
+-------------+---------------+-----------+---------------+----------+--------+
|      599730 | mysql         | a2c199    | mysql-backup  | MySQL    | READY  |
+-------------+---------------+-----------+---------------+----------+--------+

```

### CODE REVIEW

File: cmd/database/database_backup.go [hunk 1]
```diff
@@ -40,5 +40,4 @@ func init() {
 	// Update cmd options
 	dbBackupUpdateCmd.Flags().StringVarP(&name, "name", "n", "", "name of the database backup")
 	dbBackupUpdateCmd.Flags().StringVarP(&schedule, "schedule", "s", "", "schedule of the database backup in the form of cronjob")
-	dbBackupUpdateCmd.Flags().IntVarP(&count, "count", "c", 0, "number of backups to keep")
```
Comment (RealHarshThakur) [inline cmd/database/database_backup.go:43]: is this breaking change? 

How can user configure this now?

**Comment by RealHarshThakur** [comment]
I think it might be worth updating README with some basic steps around creating and restoring backups

File: cmd/database/database_backup.go (hunk 1 above)
Comment (vishalanarase) [reply cmd/database/database_backup.go:43]: we support 1 scheduled backup by default from backend

Comment (RealHarshThakur) [reply cmd/database/database_backup.go:43]: And if they want to store more?

Comment (vishalanarase) [reply cmd/database/database_backup.go:43]: It's free so we support only one.

Comment (vishalanarase) [reply cmd/database/database_backup.go:43]: They can create manual backups which chargeable

File: doc/DTABASE_BACKUP_RESTORE.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+The restore process allows you to recover your database from a previously created backup.
+
+* Support: PostgreSQL, MySQL
+
+# CLI Operations
+
+## Create MySQL and PostgreSQL database
+
+```bash
+❯ ./civo --config staging.civo.json database create postgres-demo --size g3.db.medium --software PostgreSQL --version 14
```
Comment (RealHarshThakur) [inline doc/DTABASE_BACKUP_RESTORE.md]: Can we change it to `civo database`. Basically, remove `./` and `--config` . So users can copy-paste commands

Comment (RealHarshThakur) [reply doc/DTABASE_BACKUP_RESTORE.md]: Also, make a section in README.md to point to this doc. 
We have sections for resources. I think it's good to have separate docs but we should just link it from README
https://github.com/civo/cli?tab=readme-ov-file#region

Comment (vishalanarase) [reply doc/DTABASE_BACKUP_RESTORE.md]: Addressed

**Review by RealHarshThakur — APPROVED** [review]

**Pull request merged**